geode-docs/getting_started/intro_to_clients.html.md.erb

```diff
@@ -0,0 +1,353 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<% set_title("Introduction to", product_name, "Clients") %>
+
+This section provides basic starting points for a variety of <%=vars.product_name %> clients, along
+with very rudimentary connect, put, get, and then a reference to more in-depth docs and examples on
+how to use the client.
+
```

Comment:
It could be a good idea to have an index with links to the different chapters here.

Reply:
Sorry, I think I did not explain it well. When I wrote my comment I was thinking about a index of this user guide article, so the reader can have an idea of the available clients from the beggining.

* Start an Apache Geode Cluster
* Java client
* SpringBoot client
* Native clients
  * .Net client
  * C++ client
* Geode APIS compatible with Redis

What do you think?

Reply:
I instructed bookbinder to generate a table of contents for a page. How does this look?
![Screen Shot 2021-06-11 at 9 14 37 AM](https://user-images.githubusercontent.com/13138489/121717178-c0e36d00-ca95-11eb-9807-124f6443b011.png)

Reply:
Looks great! Im curious about how bookbinder is generating that because I just see you added the anchors to each section name but it seems there is no code related with the index itself.

Reply:
This is the default behavior for bookbinder. You will see in many other doc sources that anchors contain code to suppress this: `class="no-quick-link"`